back-office/shop-parameters/customer-settings/groups.md

```diff
@@ -0,0 +1,174 @@
+- Discount (%)<br>
+- Members<br>
+- Show prices<br>
+- Creation date<br>
+
+To allow users to narrow down their search, searching and/or sorting options are available according to the columns:
+
+Input searching fields are available for ID, Group name, Discount (%), Members
+Drop-down with 2 selectable options (Yes, No) is available for Show prices column
+Date pickers (From and to) are available for Creation date column
+Sorting ASC & DESC are available for ID, Group name, Discount (%), Members and Creation date columns
+
+At the beginning of each line is a checkbox while, at the end of each line, 3 actions can be done: _Edit_, _View_, and _Delete_. The main action is _Edit_. The _View_ action can be performed by clicking on the line.
+Checkboxes play with the 'Bulk actions' selector that allows either to 'Select all', 'Unselect all', or 'Delete selected'.
+
+PrestaShop's 3 default groups cannot be deleted so they have no checkbox nor _Delete_ actions.
+
+
+## Default groups options
+
+Users can select which group designates the 3 following customers profiles:
+
+**Visitors group.** By default, it is 'Visitor' but merchants can use the dropdown selector to choose any group. A tooltip is available: `The group defined for your un-identified visitors.` in `Admin.Shopparameters.Help`.
+
+**Guests group.** By default, it is 'Guest' but merchants can use the dropdown selector to choose any group. A tooltip is available: `The group defined for your identified guest customers (used in guest checkout).` in `Admin.Shopparameters.Help`.
+
+**Customers group.** By default, it is 'Customer' but merchants can use the dropdown selector to choose any group. A tooltip is available: `The group defined for your identified registered customers.` in `Admin.Shopparameters.Help`.
+
+Users should click 'Save' to update their default groups' settings. If all settings are properly saved, then a successful message is displayed: `The settings have been successfully updated.` in `Admin.Notifications.Success`.
+
+
+## Multistore behavior
+
+### Specific shop or group context
+
+In a specific shop context or in a group context, the options listed above should have [checkboxes](https://github.com/PrestaShop/PrestaShop/issues/19366).
+
+
+### All shops context
+
+In an all shops context, the options listed above should have [specific settings drop-down](https://github.com/PrestaShop/PrestaShop/issues/19318).
+
+:point_right: https://github.com/PrestaShop/prestashop-specs/blob/b912faba2d354ec41851fd37b5c97e89350a3b79/back-office/multistoregeneralspecs.md
+
+
+## Add form
+
+Clicking 'Add new group' redirects to a form.
+
+**Name.** Users must **type the name of the customers group**. By default, it is limited to 32 characters. If the merchant tries to save a name exceeding this number, then he should stay on this page and get an error notification right below the field: `This field cannot be longer than %limit% characters (incl. HTML tags).` in `Admin.Notifications.Error`.
```

Comment:
You're right, @JevgenijVisockij, it would be clearer to mention the field type directly after its name. I usually specify it in the following paragraph but that could definitely be improved! I'll add it to the checklist and keep it in mind for the next time.